x/ecocredit/server/testsuite/suite.go

```diff
@@ -618,45 +621,61 @@ func (s *IntegrationTestSuite) TestScenario() {
 		})
 	}
 
-	/****   TEST SET PRECISION   ****/
-	precisionCases := []struct {
-		name string
-		msg  ecocredit.MsgSetPrecision
-		ok   bool
+	/****   TEST CREDIT TYPES   ****/
+	creditTypeCases := []struct {
+		name        string
+		creditTypes []*ecocredit.CreditType
+		tx          ecocredit.MsgCreateClass
```

Comment:
```suggestion
		msg         ecocredit.MsgCreateClass
```

This isn't really a transaction!

Reply:
ah, i use the two interchangeably - will switch to msg if that makes more sense here